packages/doenetml-worker-rust/lib-doenetml-core/src/core/core.rs

```diff
@@ -35,14 +37,15 @@ impl Core {
     ///
```

Comment:
Add some description of what the return value is. (It apparently is used only in tests.)